src/pages/dashboard/pending-unstake-panel/pending-unstake-panel.tsx

```diff
@@ -77,55 +76,47 @@ const PendingUnstakePanel = (props: Props) => {
   };
 
   return (
-    <BorderedBox
-      header={
-        <Header alignCenter>
-          <p className={styles.pendingUnstakeTitle}>Pending API3 tokens unstaking</p>
-        </Header>
-      }
-      content={
-        <div className={styles.pendingUnstakeContent} data-cy="pending-unstake">
-          <div className={classNames(styles.pendingUnstakeRow, styles.amount)}>
-            <p className={styles.pendingUnstakeName}>Amount</p>
-            <h5 data-cy="amount">{formatAndRoundApi3(amount)}</h5>
-          </div>
-          <div className={classNames(styles.pendingUnstakeRow, { [globalStyles.tertiaryColor]: canUnstake })}>
-            <p className={styles.pendingUnstakeName}>Cooldown</p>
-            <div className={styles.pendingUnstakeCountdown}>
-              <div className={styles.pendingUnstakeCountdownItem}>
-                <p className={classNames(globalStyles.textSmall, globalStyles.medium)}>{timerDays}</p>
-                <p className={styles.shortcut}>D</p>
-              </div>
-              <div className={styles.colon}>:</div>
-              <div className={styles.pendingUnstakeCountdownItem}>
-                <p className={classNames(globalStyles.textSmall, globalStyles.medium)}>{timerHours}</p>
-                <p className={styles.shortcut}>HR</p>
-              </div>
-              <div className={styles.colon}>:</div>
-              <div className={styles.pendingUnstakeCountdownItem}>
-                <p className={classNames(globalStyles.textSmall, globalStyles.medium)}>{timerMinutes}</p>
-                <p className={styles.shortcut}>MIN</p>
-              </div>
-              <div className={styles.colon}>:</div>
-              <div className={styles.pendingUnstakeCountdownItem}>
-                <p className={classNames(globalStyles.textSmall, globalStyles.medium)}>{timerSeconds}</p>
-                <p className={styles.shortcut}>SEC</p>
-              </div>
+    <div className={styles.pendingUnstakePanel}>
+      <p className={styles.pendingUnstakeTitle}>Pending API3 tokens unstaking</p>
+      <div className={styles.pendingUnstakeContent} data-cy="pending-unstake">
+        <div className={classNames(styles.pendingUnstakeRow, styles.amount)}>
+          <p className={styles.pendingUnstakeName}>Amount</p>
+          <h5 data-cy="amount">{formatAndRoundApi3(amount)}</h5>
+        </div>
+        <div className={classNames(styles.pendingUnstakeRow, { [globalStyles.tertiaryColor]: canUnstake })}>
+          <p className={styles.pendingUnstakeName}>Cooldown</p>
+          <div className={styles.pendingUnstakeCountdown}>
+            <div className={styles.pendingUnstakeCountdownItem}>
+              <p className={classNames(globalStyles.textSmall, globalStyles.medium)}>{timerDays}</p>
+              <p className={styles.shortcut}>D</p>
+            </div>
```

Comment:
You should remove all global styles from here, they are either unnecessary or incorrect
```suggestion
        <div className={styles.pendingUnstakeRow}>
          <p className={styles.pendingUnstakeName}>Cooldown</p>
          <div className={styles.pendingUnstakeCountdown}>
            <div className={styles.pendingUnstakeCountdownItem}>
              <p>{timerDays}</p>
              <p className={styles.shortcut}>D</p>
            </div>
```

Reply:
Did that and also fine-tuned the counter box a bit, it was not aligned properly with the amount

![image](https://github.com/user-attachments/assets/c6ce2f10-3b35-4713-878a-a7aa002cb944)